app/gui/src/project-view/components/visualizations/TableVisualization.vue

```diff
@@ -64,7 +69,7 @@ interface Error {
   all_rows_count?: undefined
 }
 
-interface ValueType {
+export interface ValueType {
```

Comment:
I feel like this should live somewhere else. At the moment it is creating a circular dependency with the .ts files.